plugin.rb

```diff
@@ -34,27 +38,17 @@ def add_user_to_groups(user, groups)
       end
 
       def send_solidarity_pm(user)
-        admin = User.find_by_username(SiteSetting.debtcollective_solidarity_message_author)
-        admin ||= Discourse.system_user
+        bloc_manager = User.find_by_username(SiteSetting.debtcollective_solidarity_message_author)
+        bloc_manager ||= Discourse.system_user
 
-        PostCreator.create(admin,
+        PostCreator.create(bloc_manager,
           archetype: Archetype.private_message,
-          title: "Solidarity Bloc",
-          raw: pm_content(user),
+          title: SiteSetting.debtcollective_solidarity_message_title,
+          raw: SiteSetting.debtcollective_solidarity_message_content % user.username,
```

Comment:
@shoshber what does the `%` do here?

Reply:
`SiteSetting.debtcollective_solidarity_message_content` is like a format template, and `user.username` is substituted into the right place. kinda like `printf` in other languages.

https://apidock.com/ruby/String/%25

Reply:
cool, I haven't used that in the past, good to know it exists. What about using the template format it provides

```ruby
"foo = %{foo}" % { :foo => 'bar' }        #=> "foo = bar"
```

It will read better and is easier to extend.

Reply:
ok